src/tests/unit/components/app-content-switcher/AppContentSwitcher.spec.jsx

```diff
@@ -0,0 +1,67 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { beforeAll, describe, expect, it, vi } from 'vitest'
+import AppContentSwitcher from '~/components/app-content-switcher/AppContentSwitcher'
+import { defaultStyles } from '~/components/app-content-switcher/AppContentSwitcher.styles'
+
+const mockOnChange = vi.fn()
+const switchOptions = {
+  left: { text: 'Text 1', tooltip: 'This is Text 1' },
+  right: { text: 'Text 2', tooltip: 'This is Text 2' }
+}
+
+const renderSwitcher = (active = false) => {
+  return render(
+    <AppContentSwitcher
+      active={active}
+      onChange={mockOnChange}
+      styles={defaultStyles}
+      switchOptions={switchOptions}
+      typographyVariant='h6'
+    />
+  )
+}
+
+describe('AppContentSwitcher', () => {
+  beforeAll(() => {
+    vi.clearAllMocks()
+  })
```

Comment:
suggest to use afterEach or remove it at all if you don't need it.

Reply:
Indeed, before there was no need